middleware.ts

```diff
@@ -4,6 +4,6 @@ import detectLanguage from './middlewares/detectLanguage';
 
 const nextMiddleware = createMiddleware(NextResponse);
 
-const { middleware, matcher } = nextMiddleware([detectLanguage]);
+export const { middleware } = nextMiddleware([detectLanguage]);
 
-export { middleware, matcher };
+export const config = { matcher: ["/"] };
```

Comment:
Note, this cannot be a re-export so its value can be statically analyzed:

From the [Next.js docs](https://nextjs.org/docs/app/building-your-application/routing/middleware#matcher):
> Good to know: The matcher values need to be constants so they can be statically analyzed at build-time. Dynamic values such as variables will be ignored.